pyprep/find_noisy_channels.py

```diff
@@ -438,6 +444,15 @@ def find_bad_by_ransac(
                 " quality tests."
             )
 
+        # Generate random channel picks for each RANSAC sample
+        self.random_ch_picks = []
+        rng = check_random_state(self.random_state)
+        for i in range(n_samples):
+            # Pick a random subset of clean channels to use for interpolation
+            good_chans = np.arange(chn_pos_good.shape[0])
```

Comment:
good_chans could be out of the loop right? I mean Im sure the impact is minimal but you know...